src/js/i18n/en-US.yml

```diff
@@ -424,9 +424,8 @@ careOptsFrontend:
         countView:
           actionsCount: "{itemCount, plural, one {# Action} other {# Actions}}"
           flowsCount: "{itemCount, plural, one {# Flow} other {# Flows}}"
-          maximumActionsCountNarrowed: Showing {itemCount, plural, one {# Action} other {# Actions}} of {maximumCount}.
-          maximumFlowsCountNarrowed: Showing {itemCount, plural, one {# Flow} other {# Flows}} of {maximumCount}.
           maximumListCount: Showing {maximumCount} of many {isFlowList, select, true {Flows} false {Actions}}.
+          maximumListCountNarrowed: Showing {itemCount} of {maximumCount} {isFlowList, select, true {Flows} false {Actions}}.
```

Comment:
We no longer need to display a plural vs. non-plural version of `"Flow"`/`"Flows"` and `"Action"`/`"Actions"`. So we can now use the `isFlowList` variable to determine if `"Actions"` or `"Flows"` is displayed.